README.md

```diff
@@ -27,18 +27,39 @@ Put jsonnet method definitions together in a directory, each named named
 server receives a call to that method.
 
 The request protobuf message is marshaled to JSON and set as the `input` jsonnet
-top-level argument when evaluating the jsonnet method. It is marshaled as:
+top-level argument when evaluating the jsonnet method. For a
+non-client-streaming method, it is marshaled as:
```

Comment:
non-client-streaming made me pause.
I'd go with unary or server-streaming. Same below.

Reply:
I re-wrote that section that described the input/output message formats. As I tried to fix it as you described, it made less sense. Please re-read this section and let me know how it reads. I have incorporated all your suggestions.